src/main/java/frc5124/robot2020/RobotContainer.java

```diff
@@ -57,15 +60,17 @@
   public static final Joystick driverRight = new Joystick(1);
   public XboxController operator = new XboxController(2);
   
-  public JoystickButton operatorA = new JoystickButton(operator, XboxController.Button.kA.value);
-  public JoystickButton operatorB = new JoystickButton(operator, XboxController.Button.kB.value);
-  public JoystickButton operatorX = new JoystickButton(operator, XboxController.Button.kX.value);
-  public JoystickButton operatorY = new JoystickButton(operator, XboxController.Button.kY.value);
-  public JoystickButton operatorLB = new JoystickButton(operator, XboxController.Button.kBumperLeft.value);
-  public JoystickButton operatorRB = new JoystickButton(operator, XboxController.Button.kBumperRight.value);
-  public JoystickButton operatorBack = new JoystickButton(operator, XboxController.Button.kBack.value);
-  public JoystickButton operatorStart = new JoystickButton(operator,XboxController.Button.kStart.value);
-  public JoystickButton operatorTest = new JoystickButton(operator, XboxController.Button.kBack.value);
+  public JoystickButton operatorA = new JoystickButton(operator, 2);
+  public JoystickButton operatorB = new JoystickButton(operator, 3);
+  public JoystickButton operatorX = new JoystickButton(operator, 1);
+  public JoystickButton operatorY = new JoystickButton(operator, 4);
+  public JoystickButton operatorLB = new JoystickButton(operator, 5);
+  public JoystickButton operatorRB = new JoystickButton(operator, 6);
+  public JoystickButton operatorLT = new JoystickButton(operator, 7);
+  public JoystickButton operatorRT = new JoystickButton(operator, 8);
+  public JoystickButton operatorBack = new JoystickButton(operator, 9);
+  public JoystickButton operatorStart = new JoystickButton(operator,10);
+  public JoystickButton operatorTest = new JoystickButton(operator, 9);
   public JoystickButton operatorStickLeft = new JoystickButton(operator, XboxController.Button.kStickLeft.value);
   public JoystickButton operatorStickRight = new JoystickButton(operator, XboxController.Button.kStickRight.value);
```

Comment:
Why the change from Xbox enum to straight numbers? I don't have a preference, but I'm curious why.
Also, lines 74 and 75 should be consistent with the decision for the rest of the constructions.

Reply:
i changed the controller mode (there's a little switch on the back of the controller). It basically makes the triggers as buttons instead of axes. with that, the values of these buttons changed, so I just set straight numbers. I agree w the changing lines 74 & 75. I believe those are buttons 11 (left) & 12 (right).